dhcpv6/ztp/parse_remote_id.go

```diff
@@ -0,0 +1,80 @@
+package ztpv6
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/insomniacslk/dhcp/dhcpv6"
+)
+
+var (
+	// Arista Port, Vlan Pattern
+	aristaPVPattern = regexp.MustCompile("Ethernet(?P<port>[0-9]+):(?P<vlan>[0-9]+)")
+	// Arista Slot, Mod, Port Pattern
+	aristaSMPPattern = regexp.MustCompile("Ethernet(?P<slot>[0-9]+)/(?P<module>[0-9]+)/(?P<port>[0-9]+)")
+)
+
+// CircuitID represents the structure of network vendor interface formats
+type CircuitID struct {
+	Slot    string
+	Module  string
+	Port    string
+	SubPort string
+	Vlan    string
+}
+
+// ZTPRemoteId will parse the RemoteId Option data for Vendor Specific data
+func ZTPRemoteId(packet dhcpv6.DHCPv6) (CircuitID, error) {
```

Comment:
How about returning a reference to the C`ircuitID` to avoid copying the whole struct?